### PR TITLE
feat: Add the logo of the built-in extension

### PR DESCRIPTION
**Pull request by yiliang114**
before: 
image

### CODE REVIEW

**Comment by rebornix** [comment]
@daviddossett do we have guidance on adding builtin extension icons?

File: extensions/notebook-renderers/media/icon.png
Comment (daviddossett) [inline extensions/notebook-renderers/media/icon.png:1]: Looks like it has a transparent background. How does this look when rendered in context?

**Comment by daviddossett** [comment]
> @daviddossett do we have guidance on adding builtin extension icons?

@rebornix nothing specific that I know of but I added some comments re: the image formats.

File: extensions/ipynb/media/icon.png
[the remark this answers is not in the document]
Comment (yiliang114) [reply extensions/ipynb/media/icon.png:1]: I actually got this image from [the microsoft/vscode-jupyter extension](https://github.com/microsoft/vscode-jupyter/blob/main/icon.png), but I did ignore the size of the file before, now I have compressed and cropped it

File: extensions/notebook-renderers/media/icon.png
Comment (yiliang114) [reply extensions/notebook-renderers/media/icon.png:1]: Here's what it looks like under the black and white themes, respectively:

image

image

In fact, this image is also from [the microsoft/vscode-notebook-renderers extension](https://github.com/microsoft/vscode-notebook-renderers/blob/main/icon.png).

**Review by daviddossett — APPROVED** [review]

**Review by aeschli — APPROVED** [review]

**Pull request merged**